clients/cobol-lsp-vscode-extension/src/ProfileService.ts

```diff
@@ -17,19 +17,25 @@ import * as path from "path";
 import * as vscode from "vscode";
 import { SETTINGS_SECTION } from "./constants";
 import { ProfilesMap, ZoweApi } from "./ZoweApi";
+import { Disposable } from "vscode-languageclient";
+
+const DEFAULT_STATUS_TEXT = "CPY profile: undefined";
```

Comment:
if it is a message better use full-text: COPYBOOKS

Reply:
Are you sure? It will be too long for the status bar.